src/controllers/routes.ts

```diff
@@ -1,12 +1,14 @@
 import { Router } from '@well-known-components/http-server'
 import { GlobalContext } from '../types'
 import { pingHandler } from './handlers/ping-handler'
+import { transportHandler } from './handlers/transport-handler'
 
 // We return the entire router because it will be easier to test than a whole server
 export async function setupRouter(_: GlobalContext): Promise<Router<GlobalContext>> {
   const router = new Router<GlobalContext>()
 
   router.get('/ping', pingHandler)
+  router.get('/transport-registration', transportHandler)
```

Comment:
is this endpoint for internal use only?

Reply:
it will be exposed outside of the catalyst so the transport it's deployed in say, amazon and register itself here

Reply:
the transport will register itself using this GET endpoint?

Reply:
yes, it's a get that gets upgraded to websocket, it's an active connection